packages/cyberstorm/src/components/PackageList/PackageList.tsx

```diff
@@ -69,32 +63,44 @@ export function PackageList(props: Props) {
     deferredDeprecated,
   ]);
 
+  const isNotStale =
```

Comment:
Nitpick: I would change this to `const isStale = deferredOrder !== order || [...]`

Using `&&` requires checking all the values, while `||` can exit early when the first changed value is detected.

It would also change `isStale={!isNotStale}` to `isStale={isStale}` which is more readable.

Reply:
Yeah, true! It was so late when I was doing this, that I couldn't figure out quickly on how to do this better, so I just made it "isXobject" styled check and flipped the variable name.

This is nice, now it seems obvious but back then just couldn't gather it from my thoughts :D